tox.ini

```diff
@@ -32,6 +32,18 @@ deps =
     mock
     mock-ssh-server
 
+[testenv:py34-unit]
+commands = bash run_tests.sh --no-create-venv -u
+whitelist_externals = bash
+setenv =
+    GALAXY_VIRTUAL_ENV = .venv3
+    GALAXY_PYTHON_VERSION=3
+deps = 
+    nose
+    NoseHTML
+    mock
+    mock-ssh-server
```

Comment:
Pipfile also for this?

Reply:
Yeah - we should just throw those all in dev-depenedences I think. I understand historically why I did it this way but it doesn't make sense anymore.